db/sqlinterfaces.c

```diff
@@ -4030,6 +4031,67 @@ static int get_prepared_stmt_int(struct sqlthdstate *thd,
         return handle_bad_transaction_mode(thd, clnt);
     }
 
+    if (clnt->is_expert) {
```

Comment:
Move this into a function and put it into `handle_non_sqlite_requests`, like the way we did for explain?